src/index.html

```diff
@@ -4,9 +4,322 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+     rel="preconnect"
+     href="https://fonts.gstatic.com"
+     crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__link">
+        <img
+             src="images/logo.png"
+             alt="moyo-logo"
+             class="logo"
+        >
```

Comment:
Need to check and fix it everywhere
```suggestion
        <img
          src="images/logo.png"
          alt="moyo-logo"
          class="logo"
        >
```